charts/argo-workflows/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.16.7
+version: 0.17.0
```

Comment:
I removed `.Values. images` , so I upgraded minor version. 
I am wondering if I should keep `.Values.images` as deprecated... 🤔